.github/workflows/go.yml

```diff
@@ -30,40 +33,33 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
-    - name: Get dependencies
-      run: |
-        go get -v -t -d ./...
-        if [ -f Gopkg.toml ]; then
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-        fi
-
     - name: Format
       run: |
-        diff -u <(echo -n) <(gofmt -d .)
+        make go-fmt
```

Comment:
By using `make` we have unified place for storing build chain definitions, so the same command is executed locally as in CI.